deploy/k8s/deployment.yaml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kubesimplify-website
+  namespace: default
+  labels:
+    app: kubesimplify
+spec:
+  selector:
+    matchLabels:
+      app: kubesimplify
+  template:
+    metadata:
+      labels:
+        app: kubesimplify
+    spec:
+      containers:
+      - name: kubesimplify-cont
+      # TODO: Replace the image with new image
+        image: public.ecr.aws/dipankardas011/kubesimplify-website:v2
```

Comment:
We need to change this to have kubesimplify and not your personal

Reply:
Got the new image 🙏

Reply:
What is the image being used?

Reply:
We do not have a kubesimplify account , need to think how can we do it and also give access to a few people

Reply:
then let this pr be in this state till the registry is ready to be used👍